inspire_json_merger/inspire_json_merger.py

```diff
@@ -135,3 +138,13 @@ def get_acquisition_source(json_obj):
         return json_obj['acquisition_source']['source']
     except KeyError:
         return 'arxiv'
```

Comment:
@michamos is this assumption still valid?

Reply:
I don't know. Do you remember why you wrote this?

Reply:
as discussed IRL, the right thing to do when `acquisition_source` is missing is assuming that the record comes from a publisher.